crates/ruff_linter/src/codes.rs

```diff
@@ -480,6 +480,7 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         (Flake8Simplify, "223") => (RuleGroup::Stable, rules::flake8_simplify::rules::ExprAndFalse),
         (Flake8Simplify, "300") => (RuleGroup::Stable, rules::flake8_simplify::rules::YodaConditions),
         (Flake8Simplify, "401") => (RuleGroup::Stable, rules::flake8_simplify::rules::IfElseBlockInsteadOfDictGet),
+        (Flake8Simplify, "905") => (RuleGroup::Preview, rules::flake8_simplify::rules::SplitOfStaticString),
```

Comment:
Nit: How about simplifying it to `SplitStaticString`?

Reply:
Will do. I inserted "of"  when I saw "to" in "static-join-to-f-string", but in hindsight that name does not even adhere to the naming convention (name the thing the rule detects)